guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>"Those items with a download, including file geodatabases, shapefiles, code samples, zip files, and packages amongst others" - Do you mean download-enabled items?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Changed it to read. I was under the impression that file types like this are automatically enabled for download. I clarified to refer to those types of items as separate for items that are enabled for export. Does this clarify?:</p><p> </p><p><em>Simple Types</em></p><ul><li><em>Those items with a download option (see&nbsp;</em><a href="https://developers.arcgis.com/rest/users-groups-and-items/items-and-item-types.htm" target="_blank"><em>Data files</em></a><em>&nbsp;for items that may be in a Web GIS and available for download), including zipped file geodatabases and shapefiles, code samples, zip files, and packages amongst others.</em></li></ul><p> </p>